geomstats/information_geometry/information_manifold.py

```diff
@@ -0,0 +1,107 @@
+"""Mixin for manifolds of probability distributions."""
+
+import geomstats.backend as gs
+import geomstats.errors
+
+
+class InformationManifold:
+    """Mixin for manifolds of probability distributions."""
+
+    def sample(self, point, n_samples=1):
+        """Sample from the probability distribution.
+
+        Sample from the probability distribution with parameters provided
+        by point. This gives n_samples points.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., dim]
+            Point representing a probability distribution.
+        n_samples : int
+            Number of points to sample for each set of parameters in point.
+            Optional, default: 1.
+
+        Returns
+        -------
+        samples : array-like, shape=[..., n_samples]
+            Sample from the probability distributions.
+        """
+        raise NotImplementedError("The sample method is not yet implemented.")
+
+    def point_to_pdf(self, point):
```

Comment:
We can simplify this (and all the inheritances then), by doing:

```python
def point_to_pdf(self, point):
    geomstats.errors.check_belongs(point, self)
    point = gs.to_ndarray(point, to_ndim=2)

   return lambda x: self.pdf(x, point=point)
```

And then each distribution needs to define `pdf(x, point)`.

This way we now `point_to_pdf` is consistent in all distributions by design.

Same applies for `cdf`.

It also simplifies handling of docstrings.

`pdf` and `cdf` don't need necessarily to be public.

Reply:
Good idea, done.